src/System.Data.SqlClient/src/System/Data/ProviderBase/DbConnectionPool.cs

```diff
@@ -1376,9 +1376,17 @@ private void PoolCreateRequest(object state)
                                 {
                                     while (NeedToReplenish)
                                     {
-                                        // Don't specify any user options because there is no outer connection associated with the new connection
-                                        newObj = CreateObject(owningObject: null, userOptions: null, oldConnection: null);
-
+                                        try
+                                        {
+                                            // Don't specify any user options because there is no outer connection associated with the new connection
+                                            newObj = CreateObject(owningObject: null, userOptions: null, oldConnection: null);
+                                        }
+                                        catch(Exception)
```

Comment:
Since you're catching everything, you can just do catch without the Exception part.

Reply:
Agreed.